### ISSUE TITLE
Please add release tags

### ISSUE THREAD

**Issue by tillea** [pauldijou/ioslides#1]
Hi,
as I Debian maintainer I need to package ioslides as a dependency of another project I need to package.  It would help a lot if you could add release tags on one hand to know what code should be distributed with Debian, when new versions are available etc and on the other hand also to inform the general public about the current version of this project.
Thanks a lot, Andreas.

**Comment by pauldijou** [comment]
This is not the official repo for Google I/O styled slides, just a personal one kind of fork. Don't you think you should use a better one? I have my personal presentations inside this repo.

**Comment by tillea** [comment]
On Tue, Dec 12, 2017 at 09:31:14PM +0000, Paul Dijou wrote:
> This is not the official repo for Google I/O styled slides, just a personal one kind of fork. Don't you think you should use a better one? I have my personal presentations inside this repo.
Could you please point me to the official one?  Thank you, Andreas.

**Comment by pauldijou** [comment]
Not really... I did a quick lookup and could not find it anymore, maybe Google removed it, it's pretty old stuff. If you still want to use this repo, I would recommend forking it on your own account and removing both `parisjs` and `zenexity` folder since those are my personal slides.
